types/open-api/thread.yaml

```diff
@@ -57,6 +61,8 @@ components:
             - id
             - parent_board_slug
             - parent_board_id
+            - parent_realm_slug
+            - parent_realm_id
```

Comment:
Not sure if these need to be required or not, need to do a lot of cleaning because Open-API didn't like me adding this

Reply:
They should be, yes